### PR TITLE
add SETS_2 code

### PR DESCRIPTION
**Pull request by MylesLJ**
add current SETS_2 Code and moved old code to folder

### CODE REVIEW

File: Test Stand/Sets_2/.DS_Store
Comment (mariano-f-r) [inline Test Stand/Sets_2/.DS_Store:1]: We can delete .DS_Store files as they are intrinsic to MacOS only

File: Test Stand/Sets_2/SETS_2 Code/.DS_Store
Comment (mariano-f-r) [inline Test Stand/Sets_2/SETS_2 Code/.DS_Store:1]: See above

File: Test Stand/Sets_2/SETS_2 Code/Old_Code/New Press Dat_Test.py
Comment (mariano-f-r) [inline Test Stand/Sets_2/SETS_2 Code/Old_Code/New Press Dat_Test.py:1]: We should avoid having spaces in file names

**Review by mariano-f-r — CHANGES_REQUESTED** [review]
Looks good except for a few extraneous things we need to clean up. Nanny duty :)

**Review by MylesLJ — COMMENTED** [review]
Deleted .DS_Store files. Lgtm

**Review by mariano-f-r — APPROVED** [review]

**Pull request merged**